content/blog/2022-07-25_Galaxy_Climate_Project_Introduction_Blog_2/index.md

```diff
@@ -0,0 +1,143 @@
+---
+
+title: "Galaxy climate to support local/regional initiatives and concrete actions to fight climate change" 
+
+tease: "Second blog post presenting the Outreachy project on Galaxy Climate"
+
+authors: "Soumya Jha, Anne Fouilloux, Jean Iaquinta"
+
+source_blog: "Galaxy Climate Outreachy Project"
+
+date: "2022-07-25"
+
+---

```

Comment:
I don't know if that has an effect, but in other posts we don't have empty lines between the fields

Reply:
can you please mention that lines? Do I need to remove the linespace from 12?

Reply:
I think I got this, have removed the spaces. Please check once! Thanks!

Reply:
I still see the spaces, maybe you need to push your changes?

Reply:
Please check this too :)

Reply:
Thanks for commit